be/src/exec/vectorized/hdfs_scanner.cpp

```diff
@@ -180,6 +176,14 @@ uint64_t HdfsScanner::exit_pending_queue() {
     return _pending_queue_sw.reset();
 }
 
+Status HdfsScanner::open_random_access_file() {
```

Comment:
could it be private? if private, name could be prefixed with _

Reply:
changed to protected